ln-gateway/Cargo.toml

```diff
@@ -26,3 +27,6 @@ tide = "0.16.0"
 tracing = "0.1.26"
 tracing-subscriber = { version = "0.3.1", features = [ "env-filter" ] }
 tokio = {version = "1.0", features = ["full"]}
+
+# FIXME: kept getting `Error: attempted to set a logger after the logging system was already initialized`
+log = "0.4.17"
```

Comment:
It seemed like tracing and the `cln_plugin` logging system were in conflict somehow and I couldn't figure out exactly how. This is kind of weird because core-lightning plugins use stdin / stdout for IO and that might have something to do with it ...